code/olc.c

```diff
@@ -1475,7 +1475,7 @@ void do_resets(CHAR_DATA *ch, char *argument)
 
```

Comment:
I think the original lines look like they're from Spanish and so these should actually translate from `... no existe` to `... does not exist`